tests/tests.py

```diff
@@ -20,6 +20,17 @@ def test_readme():
                     assert tocomp in readme, tocomp
 
 
+def test_r_stata_match():
+    cmd = f"Rscript code/models/test_that_r_coefs_match_stata.R"
+    try:
+        subprocess.run(shlex.split(cmd), check=True)
+    except:
+        raise AssertionError(
+            f"""Coefficients in stata do not match replications in projection code.
+            """
+        )
```

Comment:
```suggestion
    cmd = "Rscript code/models/test_that_r_coefs_match_stata.R"
    try:
        subprocess.run(shlex.split(cmd), check=True)
    except CalledProcessError as err:
        raise AssertionError("Coefficients in stata do not match replications in projection code.") from err
```

Reply:
I'll just make this change (prob not needed but a little more precise (blanket except clauses can be dangerous if things break for reasons other than the one you were trying to catch)

Reply:
actually, since the R code already prints a useful message, I'm just allowing the call to `subprocess.run` to fail as it normally would, rather than catching the exception and printing a new error message